solutions/src/main/java/ru/sberbank/school/task02/Solution.java

```diff
@@ -0,0 +1,48 @@
+package ru.sberbank.school.task02;
+
+import java.math.BigDecimal;
+import java.util.Optional;
+
+import ru.sberbank.school.task02.exception.FxConversionException;
+import ru.sberbank.school.task02.util.Beneficiary;
+import ru.sberbank.school.task02.util.ClientOperation;
+import ru.sberbank.school.task02.util.ExternalQuotesServiceDemo;
+import ru.sberbank.school.task02.util.Symbol;
+
+public class Solution {
+    public static void main(String[] args) {
+
+        CalculatorFactory calculatorFactory = new CalculatorFactory();
+        // получаем наш валютный калькулятор
+        FxConversionService calculator =
+                calculatorFactory.getFxConversionService(new ExternalQuotesServiceDemo());
+        ExtendedFxConversionService extendedCalculator =
+                calculatorFactory.getExtendedFxConversionService(new ExternalQuotesServiceDemo());
+
+        Beneficiary beneficiary = System.getenv("Beneficiary") == "CLIENT" ?
+                Beneficiary.CLIENT : Beneficiary.BANK;
+
+        try {
+            BigDecimal priceUSD = calculator.convert(ClientOperation.BUY,
+                    Symbol.USD_RUB, new BigDecimal(100));
+            System.out.println("USD: " + priceUSD);
+
+            Optional<BigDecimal> priceRUB = extendedCalculator.convertReversed(ClientOperation.BUY, Symbol.USD_RUB,
+                    new BigDecimal(100), beneficiary);
+            if (priceRUB.isPresent()) {
+                System.out.println("RUB " + priceRUB.get());
+            }
```

Comment:
&#39;}&#39; at column 13 should be on the same line as the next part of a multi-block statement (one that directly contains multiple blocks: if/else-if/else, do/while or try/catch/finally).

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a233546488>*